test/application/rooms.js

```diff
@@ -0,0 +1,25 @@
+var assert = require('assert');
+var room;
+
+describe('Rooms features', function () {
+	describe('Rooms', function () {
+		describe('Rooms.get', function () {
+			it('should be a function', function () {
```

Comment:
Actually, you might like the Mocha TDD interface:
TDD

<quote>The "TDD" interface provides suite(), test(), suiteSetup(), suiteTeardown(), setup(), and teardown().</quote>

As opposed to <quote>describe(), it(), before(), after(), beforeEach(), and afterEach()</quote>